components/post_view/message_attachments/action_button.test.jsx

```diff
@@ -18,11 +18,17 @@ describe('components/post_view/message_attachments/action_button.jsx', () => {
         expect(wrapper).toMatchSnapshot();
     });
 
+    baseProps.action.style = {border: 'none'};
+    test('styles should work', () => {
+        const wrapper = shallow(<ActionButton {...baseProps}/>);
```

Comment:
To avoid affecting other tests:

```suggestion
        const props = {
            ...baseProps,
            action: {...baseProps.action, style: {border: 'none'}},
        };
        const wrapper = shallow(<ActionButton {...props}/>);
```

Reply:
Makes sense.